packages/core-transaction-pool/lib/utils/dynamicfee-matcher.js

```diff
@@ -28,12 +32,18 @@ module.exports = transaction => {
     if (fee >= minFeeBroadcast) {
       broadcast = true
       logger.debug(
-        `Transaction ${id} eligible for broadcast (fee=${fee} >= min=${minFeeBroadcast})`,
+        `Transaction ${id} eligible for broadcast - fee of ${formatArktoshi(
+          fee,
+        )} is ${
+          fee === minFeeBroadcast ? 'equal too' : 'greater than'
```

Comment:
s/too/to/

Reply:
ugh, sorry. can someone fix that and line 60? also line 59 is missing the `is`.

Reply:
#1484